src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/ServiceConfigTest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.config;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.config.ServiceConfigProvider;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@IntegrationTest
+class ServiceConfigTest {
+
+    private static final String TEST_SERVICE_NAME = "bulkscan";
+
+    @Autowired
+    private ServiceConfigProvider serviceConfigProvider;
+
+    @Test
+    public void serviceConfigItem_should_store_form_type_to_ocr_field_name_mappings() {
+        ServiceConfigItem configItem = serviceConfigProvider.getConfig(TEST_SERVICE_NAME);
+
+        //form types and field names are defined in application-integration.yaml
+        assertThat(configItem.getSurnameOcrFieldName("form1")).isEqualTo("lastName");
+        assertThat(configItem.getSurnameOcrFieldName("form2")).isEqualTo("surname");
```

Comment:
Would be nice to somehow clarify that the first one is being used (config has to fields for `form2`).
Perhaps even an assertion that error got logged.

Reply:
or actually, maybe better do that in the unit test

Reply:
There is a unit test for multiple mappings